acceptance/cmd/workspace/create-scope/output.txt

```diff
@@ -0,0 +1,27 @@
+
+>>> musterr [CLI] secrets create-scope --scope-backend-type AZURE_KEYVAULT --json "scope": "xxxxxxxx-xxx"
+Error: error decoding JSON at (inline):1:8: unexpected additional content
+
+Exit code (musterr): 1
+
+>>> musterr [CLI] secrets create-scope --scope-backend-type AZURE_KEYVAULT --json ["scope"]
+Error: expected JSON input of type *workspace.CreateScope, received sequence
```

Comment:
Including the SDK typename here is not really actionable to uninitiated users.

Could we say instead something like "expected a JSON object", and (optionally, if we can) refer to the REST API docs for the right structure?

Reply:
Sure, fixed here: https://github.com/databricks/cli/pull/3398/commits/f9192cab24fc513dcdf09de517096007a2517074